pwa/src/components/applicationCard/ApplicationCard.tsx

```diff
@@ -28,7 +29,10 @@ export const ApplicationCard: React.FC<ApplicationCardProps> = ({ title, descrip
   return (
     <div className={styles.container}>
       <div className={styles.titleLink} onClick={() => navigate(title.href)}>
```

Comment:
Alle links moeten `<Link href={something}>` hebben en niet meer die `onClick`, dat is niet toegankelijk.